services/QuillLMS/spec/controllers/admins_controller_spec.rb

```diff
@@ -182,12 +182,10 @@
   end
 
   describe '#approve_admin_request' do
-    before do
-      create(:admin_info, user: teacher, approval_status: AdminInfo::PENDING)
-    end
-
     it 'should update the teacher admin info to be approved, create a school admin record, call the analytics worker, and return a message' do
-      expect(TeacherApprovedToBecomeAdminAnalyticsWorker).to receive(:perform_async).with(teacher.id)
+      admin_info = create(:admin_info, user: teacher, approval_status: AdminInfo::PENDING)
+      admin_approval_request = create(:admin_approval_request, admin_info: admin_info, requestee: admin, request_made_during_sign_up: [true, false].sample )
```

Comment:
Given the use of the `[true, false].sample` pattern, I wonder if it might make sense to put that in the AdminInfo factory instead of setting it on each `create` call.